src/ugeneui/src/main_window/SplashScreen.cpp

```diff
@@ -86,7 +86,9 @@ SplashScreenWidget::SplashScreenWidget() {
     setObjectName("splash_screen_widget");
 
     Version v = Version::appVersion();
-    version = v.text;
+    version = QString::number(v.major) +
+              (v.minor == 0 ? "" : "." + QString::number(v.minor)) +
+              (v.suffix.isEmpty() ? "" : "-" + v.suffix);
```

Comment:
You can make this code a method of the Version class

Reply:
It is very specific of the SplashScreen UX that we do not want to show 0. 
Otherwise the Version class already has v.text and other places show the version like v.text.